.gitignore

```
@@ -2,4 +2,5 @@
 /winsocat/bin/
 /Tests/obj
 /Tests/bin
+/.vs
```

You can also add `.idea` folder here to prevent future additions as well. 
```suggestion
/.vs/
/.idea/
```